index.html

```diff
@@ -15,28 +15,38 @@
     <header class="header">
         <a href="#" class="logo">Alpha Portifolio</a>
         <nav><i class="fa fa-bars"></i></nav>
+        <ul class="header-links">
```

Comment:
resolved 👍

Reply:
@Kingstalux, I created a branch to desktop version yesterday to complete the exercise of animations before the approval and then merge to main. Because of that, the pull request of the desktop version project has been marked closed. Is it fine to share this exact pull request link that has been closed? or do I have to create a new pull request?